### ISSUE TITLE
19.10: Search in Activities overview does not return any results

### ISSUE THREAD

**Issue by brs17** [pop-os/pop#666]
**Distribution (run `cat /etc/os-release`):**
 Pop!_OS 19.10
 
 
**Issue/Bug Description:**
When searching for a program in the Activities overview, no results are returned.

**Comment by brs17** [comment]
As it turns out, searching for something like `firefox` and then pressing enter will launch `firefox`. So I believe the results of the search are just not shown. I am able to confirm that by typing incomplete program names such as `fire` or `pop`.

**Comment by isantop** [comment]
I'm fairly sure this is a theme bug, as changing to the Yaru shell theme causes results to be displayed.

**Comment by mmstick** [comment]
Indeed, it is an issue in the shell theme. Changing the stylesheet for the Pop session's mode from `pop.css` to Yaru fixes it.
